src/components/animatedScanner/index.js

```diff
@@ -103,14 +103,15 @@ export default class AnimatedScanner extends BaseComponent {
   }
 
   renderNew() {
-    const {opacity, backgroundColor, hideScannerLine} = this.props;
+    const {opacity, backgroundColor, borderRadius, hideScannerLine} = this.props;
```

Comment:
I think it's preferable to allow passing `style` instead of exposing specific prop. 
I guess the thumb rule for whether to use a `style` or an explicit prop is if we're using the prop's value for internal component logic. 
In this case, since it's just cosmetics, let's use `style`.